.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+name: release
+
+on:
+  push:
+    branches:
+      - master
```

Comment:
I'm not sure if there could be a race condition between this and the build workflow. If they're not in the same workflow, I'm not sure you can guarantee order.

Combining build and release into a single workflow with two jobs and a `needs:` clause will eliminate this.

Reply:
:thinking: not really sure we want automatic releases anyway. This might be better as a manual job.

Reply:
Ah yeah this is what we want https://developer.github.com/v3/repos/deployments/

Reply:
If we don't want full automatic releases, maybe we can have a protected "release" branch that's specifically for CD. Only pushes to that branch will trigger a deploy to Fly.

Reply:
Let's table this for now. Pushes releases manually via flyctl is fine for now.

Reply:
I'm going to delete this file.